pandas/core/reshape/concat.py

```diff
@@ -619,17 +620,16 @@ def _make_concat_multiindex(indexes, keys, levels=None, names=None) -> MultiInde
         codes_list = []
 
         # things are potentially different sizes, so compute the exact codes
-        # for each level and pass those to MultiIndex.from_arrays
-
+        # for each level and pass those to MultiIndex.from_arrays.
```

Comment:
could you please revert the change here? ideally only change the code where needs to be changed